weblab_cg/data/tests/chaste_reference_models/Normal/hodgkin_huxley_squid_axon_model_1952_modified.hpp

```diff
@@ -0,0 +1,77 @@
+#ifndef DYNAMICHODGKIN_HUXLEY_SQUID_AXON_MODEL_1952_MODIFIEDFROMCELLML_HPP_

```

Comment:
This file is already in the `data` directory. We don't need to have it twice

Reply:
(Sorry, I meant the cellml file)

Reply:
oeps, gone next update!